crowbar_framework/app/models/node_object.rb

```diff
@@ -19,6 +19,13 @@
 require 'timeout'
 
 class NodeObject < ChefObject
+  API_REACHABLE_STATES = [
+    "applying", "discovered", "discovering", "hardware-install",
```

Comment:
Sorry,  I keep discovering new things in this list :-) Can we remove `"hardware-install"`?